[ADD] ecommerce: new page on pricing management (#2952)
odoo/documentation: content/applications/websites/ecommerce/getting_started/pricing_management.rst

==================
Pricing management
==================

> Odoo offers multiple options to select the price displayed on your website, as well as

Reviewer: When I read this, not sure about which price we are talking about

condition-specific prices based on selected criteria.

Tax display strategy
====================

> The choice of tax display usually depends on a country's regulations or the type of customers **(B2B

Reviewer: ```suggestion
The choice of tax display usually depends on the country's regulations or the type of customers **(B2B
```

> vs B2C)**. To select the type of price displayed, go to :menuselection:`Website --> Configuration

Reviewer: ```suggestion
vs B2C)**. To select the type of price displayed, go to :menuselection:`Website app --> Configuration
```

> --> Settings`, scroll-down to the :guilabel:`Pricing` category, and select :guilabel:`Tax-Excluded`

Reviewer: ```suggestion
--> Settings`, scroll down to the :guilabel:`Pricing` category, and select :guilabel:`Tax-Excluded`
```

or :guilabel:`Tax-Included`.

Note that this setting is **global**, and the tax display type is the same across all website(s) and
back-end. It is therefore not possible to select different tax displays for different websites. This
may be a significant point of consideration when implementing a database with multiple ecommerce
websites pertaining to different types of customers (for instance, B2B and B2C).

- :guilabel:`Tax-excluded`: the price displayed on the website is **tax-excluded** and the tax is
  computed at the cart review step;
- :guilabel:`Tax-included`: the price displayed on the website is **tax-included**.

Price management
================

There are multiple ways to manage the pricing of your products.

Pricelists
----------

Pricelists are the main tool to manage prices on your ecommerce. They allow you to define specific

> prices based according to: **country groups**, **currency**, **minimum quantity**, or **variants**.

Reviewer: ```suggestion
prices according to: **country groups**, **currency**, **minimum quantity**, or **variants**.
```

Pricelists can be selected by your customer on the product's page in the upper-right corner. You can

> create as many pricelist as needed, but it is mandatory to have at least one pricelist available on

Reviewer: ```suggestion
create as many pricelists as needed, but it is mandatory to have at least one pricelist available on
```

a website. If no custom pricelists are added, Odoo defaults to the **Public Pricelist** for all
websites.

To create or configure a pricelist, go to :menuselection:`Website backend --> Products -->

> Pricelists`. You can change the :guilabel:`Name` of the pricelist, the :guilabel:`Currency`, the

Reviewer: ```suggestion
Pricelists`. You can change the :guilabel:`Pricelist Name`, the :guilabel:`Currency`, the
```

> :guilabel:`Company` in which it is available, and the :guilabel:`Price Rules`. When modifying or

Reviewer: ```suggestion
:guilabel:`Company` in which it is available. When modifying or
```

creating a line under :guilabel:`Price Rules`, you can:

.. note::

>    To offer **multiple currencies** to your customer, make sure to activate the
>    :guilabel:`Multi-Currencies` option in :menuselection:`Website backend --> Configuration -->
>    Settings` under the category :guilabel:`Pricing`.

Reviewer: ```suggestion
Activate the :guilabel:`Multi-Currencies` option to offer a multi-currencies environment to your customers. Go to the :menuselection:`Website backend ‣ Configuration ‣ Settings` and under the :guilabel:`Pricing` category check the :guilabel:`Multi-Currencies` box.
```

> - :guilabel:`Apply On`: choose to what type the pricelist applies (all products, product category,
>   single product, or product variant(s));

Reviewer: ```suggestion
- :guilabel:`Apply On`: choose to what type the pricelist applies (All Products, Product Category,
  Product, or Product Variant;
```

> - :guilabel:`Compute Price`: how the price should be computed (fixed price, percentage as discount,
>   or a formula of your choice);

Reviewer: ```suggestion
- :guilabel:`Compute Price`: how the price should be computed (Fixed Price, Percentage discount,
  or Formula)
```

- :guilabel:`Min. Quantity`: the minimum quantity at which the :guilabel:`price` is applicable;

> - :guilabel:`Fixed` or :guilabel:`Percentage Price`: the fixed price of the product (or variant) or

Reviewer: I don't see the Percentage price option

  the percentage price;

> - :guilabel:`Start` and :guilabel:`End Date`: the dates during which the pricelist is available.

Reviewer: ```suggestion
- :guilabel:`Start Date` and :guilabel:`End Date`: the dates during which the pricelist is available.
```

.. image:: pricing_management/pricing-pricelist.png
   :align: center
   :alt: Pricelist options

Under the :guilabel:`Configuration` tab, you can:

.. _pricing_management/pricelists-availability:

- :guilabel:`Country Groups`: select the country groups in which the pricelist is available;
- :guilabel:`Website`: the websites on which the pricelist is available;
- :guilabel:`Selectable`: whether or not the customer can select that pricelist;
- :guilabel:`E-commerce Promotional Code`: whether the pricelist should be applied using a promo
  code upon checkout.

.. image:: pricing_management/pricing-configuration.png
   :align: center
   :alt: Pricelist configuration

Add multiple currencies
~~~~~~~~~~~~~~~~~~~~~~~

You can assign a specific currency to a pricelist and therefore offer **multiple currencies** to

> your customers on the ecommerce. In order for your customers to be able to switch currencies, make

Reviewer: ```suggestion
your customers on the eCommerce. In order for your customers to be able to switch currencies, make
```
